src/help.json

```diff
@@ -1,41 +1,48 @@
 [
   {
-    "names": ["!help", "!h"],
+    "names": ["help", "h"],
     "role": "@everyone",
+    "usage": "!h",
```

Comment:
I would have another usage show how you can use it with an argument as well. So the `usage` property would be an array